<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>5</width>
      <height>5</height>
     </size>
    </property>
    <property name="text">
     <string>Shop by Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mens Wear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Womens Wear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Checkout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mens_Wear">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Mens_Wear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="womens">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>womens</string>
    </property>
   </widget>
   <widget class="QLabel" name="mens_image_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>91</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>Mens Wear</string>
    </property>
   </widget>
   <widget class="QLabel" name="womens_image_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>101</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>Womens Wear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBrowsing">
    <property name="title">
     <string>Browsing</string>
    </property>
   </widget>
   <addaction name="menuBrowsing"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
